baker/siteRenderers.tsx

```diff
@@ -414,6 +414,7 @@ ${dataInsights
                     homepageMetadata: get(post, "homepageMetadata", {}),
                     latestWorkLinks: get(post, "latestWorkLinks", []),
                     linkedChartViews: get(post, "linkedChartViews", {}),
+                    tags: get(post, "tags", []) ?? [],
```

Comment:
We already pass the default value (`[]`) to `get`, so we don't need the `?? []`.